### PR TITLE
Update execute.bzl

### PR DESCRIPTION
**Pull request by ahumesky**
Make `merge_tool` a private attribute (`"_merge_tool"`), like it says it is in the comments.

### CODE REVIEW

**Comment by c-parsons** [comment]
Failures look legitimate, PTAL?

**Comment by ahumesky** [comment]
yeahhh, looks like making it a public attr was on purpose to support different platforms via a select.... i guess the right thing to do is to create a wrapper macro to create a default select...

**Comment by ahumesky** [comment]
restructured things a little, PTAL

File: rules/actions_run/execute.bzl [hunk 1]
```diff
@@ -30,7 +38,19 @@ concat = rule(
             executable = True,
             cfg = "host",
```
Comment (katre) [inline rules/actions_run/execute.bzl:39]: Also, since you're here, please change this to `cfg = "exec"`.

**Comment by aiuto** [comment]
Friendly ping. Trying merge or close.

**Comment by aiuto** [comment]
Friendly ping. Trying merge or close.

**Comment by aiuto** [comment]
Last ping before closing it.

**Comment by aiuto** [comment]
One final, last ping.  If we are not going to work on this, let's close it.

**Comment by ahumesky** [comment]
I don't really recall what I was doing here, doesn't seem super important

**Pull request closed without merging**